src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPoolWorkQueue.cs

```diff
@@ -1803,6 +1879,17 @@ internal static IEnumerable<object> GetQueuedWorkItems()
                 yield return workItem;
             }
 
+#if CORECLR
+            if (ThreadPoolWorkQueue.s_prioritizationTest)
+            {
+                // Enumerate low-priority global queue
+                foreach (object workItem in s_workQueue.lowPriorityWorkItems)
+                {
+                    yield return workItem;
+                }
+            }
+#endif
```

Comment:
Can we add a test somewhere that turns on the switch and does a bit of work to just smoke test that nothing blows up?